l10n_it_pos_fatturapa_send_directly/models/pos_order.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, api
+
+
+class Order(models.Model):
+    _inherit = "pos.order"
+
+    @api.model
+    def create_from_ui(self, orders):
+        order_ids = super(Order, self).create_from_ui(orders)
+        for order in self.browse(order_ids):
+            if (
+                order.invoice_id and
+                order.invoice_id.state in ("open", "in_payment", "paid")
+            ):
+                wizard = self.env["wizard.export.fatturapa"].with_context(
+                    active_id=order.invoice_id.id, active_ids=order.invoice_id.ids,
+                    active_model="account.invoice"
+                ).create({})
+                wizard.exportFatturaPA()
+                order.invoice_id.fatturapa_attachment_out_id.send_via_pec()
```

Comment:
Se fosse abilitato un altro canale di invio? Sarebbe bello se richiamasse un metodo da implementare a seconda del canale utilizzato, cosa ne pensi?

Reply:
In generale sarebbe bene, ma questo implica una ristrutturazione di `l10n_it_sdi_channel` che al momento non mi sembra in corso nè prevista.

Gestisci canali diversi in tue implementazioni?

Reply:
Si, in particolare con DocEasy. 
Per quel che riguarda il modulo `l10n_it_sdi_channel` c'è una issue aperta https://github.com/OCA/l10n-italy/issues/2477

Reply:
@eLBati se fosse possibile, il suggerimento sarebbe creare una funzione all'interno di `pos_order`  in cui viene effettuata la chiamata a

`order.invoice_id.fatturapa_attachment_out_id.send_via_pec()` 

qualcosa del tipo:

```
self send_fatturapa(self, order):
    order.invoice_id.fatturapa_attachment_out_id.send_via_pec()
````

In questo modo si potrebbe fare override del metodo e utilizzare un altro canale per l'invio. Cosa ne pensi?

Reply:
Certo, fatto

Reply:
Grazie!